platform-coop-toolkit/coop-library
# fix: only return formatted date if it's set

## Description

For resources without publication dates, don't show the publication date.

## Steps to test

1. Find a resource that doesn't have a publication date set.
2. View it in card and single resource views.

**Expected behavior:** Publication date is not shown.

## Additional information

Not applicable.

## Related issues

Not applicable.
